packages/core/src/Page/index.js

```diff
@@ -1165,6 +1166,8 @@ class Page {
   addLayoutScriptsAndStyles() {
     this.asset.layoutScript = path.join(this.pageConfig.layoutsAssetPath, this.layout, 'scripts.js');
     this.asset.layoutStyle = path.join(this.pageConfig.layoutsAssetPath, this.layout, 'styles.css');
+    this.includedFiles.add(path.join(process.cwd(), LAYOUTS_FOLDER_PATH, this.layout, 'scripts.js'));
```

Comment:
this is the "right" fix, but `process.cwd()` wont work when you run markbind to serve a site inside a folder
https://markbind.org/userGuide/cliCommands.html#serve-command

you want `this.rootPath`